config/locales/en.yml

```diff
@@ -65,6 +65,11 @@ en:
   realistic_job_previews:
     url_apply_again: https://teacherselect.qualtrics.com/jfe/form/SV_4VHMkFk6TeonVeS
   page_titles:
+    continuous_applications:
+      your_application:
+        zero: Your application
+        one: Your application
+        other: Your applications
```

Comment:
```suggestion
        many: Your applications
```

Reply:
Just to be documented many got discarded from Rails. Now we need to use other key. :)